SECURITY.md

```diff
@@ -0,0 +1,79 @@
+# Security Policy
+
+This policy is adapted from the policies of the following CNCF projects:
+
+* [Rook](https://github.com/rook/rook)
+* [Containerd](https://github.com/containerd/project)
+
+## Reporting a Vulnerability
+
+To report a vulnerability, either:
+
+1. Report it on Github directly you can follow the procedure described
+   [here](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+   and:
+    - Navigate to the [security tab](https://github.com/crossplane/crossplane/security) on the repository
+    - Click on 'Advisories'
+    - Click on 'Report a vulnerability'
+
+2. Send an email to `crossplane-security@lists.cncf.io` detailing the issue and steps
```

Comment:
This may be very rigid, but do you only have to detail the issue and steps to reproduce if you send an email? 

I suggest detailing in a separate section what to include in the report and not as part of where to send the report.

Reply:
Thanks for the suggestion @AdamKorcz! I tried to add it in a separate section, feel free to give it a look and suggest any other detail that you feel might be worth mentioning!